modules/ROOT/pages/config-workflow.adoc

```diff
@@ -18,3 +18,8 @@ endif::[]
     * Register Mule runtimes
 
 After completing the steps in this workflow, you can deploy and manage applications, register APIs and use the API Exchange on an Anypoint Platform environment running on your own local network.
+
+[NOTE]
+At the time of this writing, some Docker containers must run as root. Current work in progress removes the need for 
+these containers to run as root. Processes running as root do not expose functionality outside the cluster. 
+Processes that serve external traffic run as non root to maximize security.
```

Comment:
Hmm...not sure we are supposed to document future functionality. Can we weasel around this?

Currently, some Docker containers must run as root.

Then we would omit the second sentence.

Reply:
non root > Processes that serve external traffic do not run as root, to maximize security.

Doesn't that contradict the previous statement?

Or "non root" > nonroot, non-root (if you think one word is confusing)

Reply:
Updated - thank you! The future functionality statement was originally put in b/c this has been (and currently is) a concern for customers. Put in the statement to tell them it's being worked on, but might as well just document it in release notes when it is truly done.